### ISSUE TITLE
Cannot listen to same event twice

### ISSUE THREAD

**Issue by sqren** [krakenjs/post-robot#12]
I am currently doing something along the lines:

```js
postRobot.on('myEventName', function() {
	// Do something
});

// In some other part of the codebase

postRobot.on('myEventName', function() {
	// Do something else
});
```

This throws `Request listener already exists for <eventName> on domain *`.

Looking in the [source](https://github.com/krakenjs/post-robot/blob/51b903a82baeb6fc91f81e85cc99d33aeac7e02a/src/drivers/listeners.js#L109), I can see it's possible to override the previous listener. 

What if I want two listeners for the same event? This is necessary keep separation of concerns.

Thanks.

**Comment by bluepnume** [comment]
@sqren Sorry I didn't see your comment until now. For some reason didn't have notifications on for this repo.

I'm confused as to how having multiple listeners with the same name would work. 

Let's say:

- I'm on window A.
- I send `foo` message to window B.
- Window B has 2 listeners for `foo`

Which listener on window B gets called?

**Comment by sqren** [comment]
@bluepnume Now it's my time to apologise for the slow reply. On a completely unrelated node I stumbled upon [this article](https://medium.com/@bluepnume/async-javascript-is-much-more-fun-when-you-spend-less-time-thinking-about-control-flow-8580ce9f73fc#.ghatk81se) and was like: wait? wasn't that the post-robot guy - who wrote a reply last week...

Aaaanyways :)

I went exactly the route you suggested above. Didn't think about the return values being an issue - makes sense.
Thanks.
